src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.scss

```diff
@@ -0,0 +1,290 @@
+/**
```

Comment:
I see a lot of the same styles are used here as the people-picker. Can they be extracted into a common CSS file and imported in both?